pyproject.toml

```diff
@@ -122,7 +123,7 @@ xclim = "xclim.cli:cli"
 "Source" = "https://github.com/Ouranosinc/xclim/"
 "Changelog" = "https://xclim.readthedocs.io/en/stable/history.html"
 "Issue tracker" = "https://github.com/Ouranosinc/xclim/issues"
-"About Ouranos" = "https://www.ouranos.ca/en/"
+"About Ouranos" = "https://ouranos.ca/en/"
 
```

Comment:
removing `www.` seems problematic in a web browser, is there a reason to do this?

Reply:
It used to be the case that `https://ouranos.ca` wouldn't resolve to our home page, but this was explicitly fixed last year. I find it more aesthetically pleasing to drop the `www.`, but I don't mind changing it back.

Reply:
right now, I enter this domain in firefox or chromium, and the page never appears, but w/ `www.` it works no problem

Reply:
Well then. Looks like the problem came back. Guess I'll resend the same email I sent last year. Will revert.